### PR TITLE
node: Serve `NetmapService.NetmapSnapshot` RPC

### PR DESCRIPTION
**Pull request by cthulhu-rider**
There is no more need to serve the same request on Control API.

* based on and blocked by https://github.com/nspcc-dev/neofs-sdk-go/pull/337

### CODE REVIEW

**Review by acid-ant — DISMISSED** [review]

**Review by fyrchik — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,10 +14,13 @@ Changelog for NeoFS Node
 ### Removed
 - Remove WIF and NEP2 support in `neofs-cli`'s --wallet flag (#1128)
 - Remove --generate-key option in `neofs-cli container delete` (#1692)
+- Serving `ControlSetvice.NetmapSnapshot` RPC (#1793)
```
Comment (carpawell) [inline CHANGELOG.md:17]:
```suggestion
- Serving `ControlService.NetmapSnapshot` RPC (#1793)
```

File: cmd/neofs-node/config.go [hunk 2]
```diff
@@ -131,6 +132,13 @@ type cfg struct {
 	persistate *state.PersistentStorage
 
 	netMapSource netmapCore.Source
+
+	// current network map
+	netMap atomicstd.Value // type netmap.NetMap
+}
+
+func (c *cfg) ProcessCurrentNetMap(f func(netmap.NetMap)) {
```
Comment (carpawell) [inline cmd/neofs-node/config.go:140]: why not just getter?

Comment (cthulhu-rider) [reply cmd/neofs-node/config.go:140]: The need is to perform exact single action. We could implement it as a getter, doesn' matter.

Comment (fyrchik) [reply cmd/neofs-node/config.go:140]: I also think `CurrentNetmap() netmap.NetMap` is simpler to read and use.

Comment (carpawell) [reply cmd/neofs-node/config.go:140]: me too

Comment (cthulhu-rider) [reply cmd/neofs-node/config.go]: Finally I've decided to leave `netmap` service package a dummy message constructor. All the business logic is implemented on application side. 0e7bc5d0cf2f29b24849d6ebb45e110f914614ed

**Review by fyrchik — APPROVED** [review]

**Review by acid-ant — APPROVED** [review]

**Pull request merged**